experimental/packages/opentelemetry-sdk-node/package.json

```
@@ -47,6 +47,9 @@
   "dependencies": {
     "@opentelemetry/api-logs": "0.52.0",
     "@opentelemetry/core": "1.25.0",
+    "@opentelemetry/exporter-logs-otlp-grpc": "^0.52.0",
+    "@opentelemetry/exporter-logs-otlp-http": "^0.52.0",
+    "@opentelemetry/exporter-logs-otlp-proto": "^0.52.0",
```

The next release PR would pin it anyway. If there's a release bugfix release of the exporter, there will also be a release of this package.

```suggestion
    "@opentelemetry/exporter-logs-otlp-grpc": "0.52.0",
    "@opentelemetry/exporter-logs-otlp-http": "0.52.0",
    "@opentelemetry/exporter-logs-otlp-proto": "0.52.0",
```